APMrover2/GCS_Mavlink.cpp

```diff
@@ -264,6 +264,33 @@ void Rover::send_fence_status(mavlink_channel_t chan)
     fence_send_mavlink_status(chan);
 }
 
+void Rover::send_wind(mavlink_channel_t chan)
```

Comment:
This should be in `GCS_MAVLINK_Rover`

Reply:
you can blame @rmackay9 for this one, I have had a quick look, all the other send_... functions are under Rover for nav controller, servos out, rangefinder, pids ect. I am happy to do whichever, thanks for having a look.